students/kegan/session03/mailroom.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python3
+
+"""
+Kathryn Egan
+
+This program manages donors and their donations for
+Miuvenile Care, a charity organizaiton for disadvantaged
+kittens.
+"""
+
+
+DONORS = {
+    'Benedict Cumberbatch': [200000.0],
+    'Elon Musk': [10000.0, 150000.0, 100000.0],
+    'Dad': [20.0, 5.0],
+    'Donald Trump': [2.81],
+    'Billy Neighbor': [.54, .01, .25]}
+
+COLUMN_WIDTHS = [0] * 4
+
+
+def main():
+    """ Module that drives main menu."""
+    options = {
+        '1': {
+            'prompt': 'Send a Thank You',
+            'module': print_thank_you},
```

Comment:
you probably want to use the word "function" or "operation" or "action", or.... "module" has a specific meaning in Python.

nice structure though!